tests/SpecTests/ClientSideEncryption/Prose22_RangeExplicitEncryptionTest.php

```diff
@@ -0,0 +1,479 @@
+<?php
+
+namespace MongoDB\Tests\SpecTests\ClientSideEncryption;
+
+use ArrayIterator;
+use Generator;
+use Iterator;
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\Decimal128;
+use MongoDB\BSON\Document;
+use MongoDB\BSON\UTCDateTime;
+use MongoDB\Driver\ClientEncryption;
+use MongoDB\Driver\Exception\EncryptionException;
+use MultipleIterator;
+
+use function base64_decode;
+use function file_get_contents;
+use function get_debug_type;
+use function is_int;
+use function version_compare;
+
+/**
+ * Prose test 22: Range Explicit Encryption
+ *
+ * @see https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#range-explicit-encryption
+ * @group csfle
+ * @group serverless
+ */
+class Prose22_RangeExplicitEncryptionTest extends FunctionalTestCase
+{
+    private $clientEncryption;
+    private $collection;
+    private $encryptedClient;
+    private $key1Id;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        if ($this->isStandalone() || ($this->isShardedCluster() && ! $this->isShardedClusterUsingReplicasets())) {
+            $this->markTestSkipped('Range explicit encryption tests require replica sets');
+        }
+
+        if (version_compare($this->getServerVersion(), '7.0.0', '<')) {
+            $this->markTestSkipped('Range explicit encryption tests require MongoDB 7.0 or later');
```

Comment:
We should create a helper `skipIfServerVersionNotSupported($version, $operator, $message)`. I create a ticket?

Reply:
SGTM. It can go on the base FunctionalTestCase class.

Reply:
https://jira.mongodb.org/browse/PHPLIB-1153